includes/EDD_SL_Plugin_Updater.php

```diff
@@ -10,333 +10,328 @@
 
-                $version_info = $this->api_request( 'plugin_latest_version', array( 'slug' => $this->slug ) );
+				set_transient( $cache_key, $version_info, 3600 );
+			}
 
-                set_transient( $cache_key, $version_info, 3600 );
-            }
 
+			if( ! is_object( $version_info ) ) {
+				return;
+			}
 
-            if( ! is_object( $version_info ) ) {
-                return;
-            }
+			if( version_compare( $this->version, $version_info->new_version, '<' ) ) {
 
-            if( version_compare( $this->version, $version_info->new_version, '<' ) ) {
+				$update_cache->response[ $this->name ] = $version_info;
 
-                $update_cache->response[ $this->name ] = $version_info;
+			}
 
-            }
+			$update_cache->last_checked = time();
+			$update_cache->checked[ $this->name ] = $this->version;
 
-            $update_cache->last_checked = time();
-            $update_cache->checked[ $this->name ] = $this->version;
+			set_site_transient( 'update_plugins', $update_cache );
 
-            set_site_transient( 'update_plugins', $update_cache );
+		} else {
 
-        } else {
+			$version_info = $update_cache->response[ $this->name ];
 
-            $version_info = $update_cache->response[ $this->name ];
+		}
 
-        }
+		// Restore our filter
+		add_filter( 'pre_set_site_transient_update_plugins', array( $this, 'check_update' ) );
 
-        // Restore our filter
-        add_filter( 'pre_set_site_transient_update_plugins', array( $this, 'check_update' ) );
+		if ( ! empty( $update_cache->response[ $this->name ] ) && version_compare( $this->version, $version_info->new_version, '<' ) ) {
 
-        if ( ! empty( $update_cache->response[ $this->name ] ) && version_compare( $this->version, $version_info->new_version, '<' ) ) {
+			// build a plugin list row, with update notification
+			$wp_list_table = _get_list_table( 'WP_Plugins_List_Table' );
+			echo '<tr class="plugin-update-tr"><td colspan="' . $wp_list_table->get_column_count() . '" class="plugin-update colspanchange"><div class="update-message">';
 
-            // build a plugin list row, with update notification
-            $wp_list_table = _get_list_table( 'WP_Plugins_List_Table' );
-            echo '<tr class="plugin-update-tr"><td colspan="' . $wp_list_table->get_column_count() . '" class="plugin-update colspanchange"><div class="update-message">';
+			$changelog_link = self_admin_url( 'index.php?edd_sl_action=view_plugin_changelog&plugin=' . $this->name . '&slug=' . $this->slug . '&TB_iframe=true&width=772&height=911' );
 
-            $changelog_link = self_admin_url( 'index.php?edd_sl_action=view_plugin_changelog&plugin=' . $this->name . '&slug=' . $this->slug . '&TB_iframe=true&width=772&height=911' );
+			if ( empty( $version_info->download_link ) ) {
+				printf(
+					__( 'There is a new version of %1$s available. <a target="_blank" class="thickbox" href="%2$s">View version %3$s details</a>.', 'edd' ),
```

Comment:
@pippinsplugins, I think the textdomains on line 179, 186 and 324 need to be changed to 'easy-digital-downloads' ?